app/v2/controllers/requestParsers/validators/CreateLossClaimValidator.scala

```diff
@@ -17,6 +17,7 @@
 package v2.controllers.requestParsers.validators
 
 import config.FixedConfig
+import v2.controllers.requestParsers.validators.validations.TypeOfLossValidation
```

Comment:
I don't think this is necessary because of line below.

Reply:
Removed.